data/HOP/data.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "HOP",
+  "name": "Hop",
   "symbol": "HOP",
   "decimals": 18,
-  "description": "A governance token",
+  "description": "A multi-chain token that powers the Hop ecosystem",
+
   "website": "https://hop.exchange",
   "nobridge": true,
```

Comment:
this token seems bridgable... can you remove this line and provide the appropriate L2 bridge address within the base subsection? (and I guess optimism too assuming it's bridgable to optimism?)

Reply:
Oh hmm, I guess your bridge may not compatible with the OptimismERC20 standard?

Reply:
If so maybe you can leave it as "nobridge" here but clarify in the description that it's "bridged by Hop network", kind of like some of the wormhole tokens do: https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/71c0a72c83c6129a6c115caa824a1abe11cafb44/data/APT/data.json#L5

Reply:
looks like our replies crossed paths, consulting with the OP labs folks there isn't a great process for non-standard bridged tokens right now, so let's just leave it as-is only please update the description to clarify it's bridged by you all.

Reply:
@roberto-bayardo sounds good. Updated the description and will wait for further instruction. Let me know if you need anything 👍